### PR TITLE
Add RKNN deploy backend support for quantized model

### PR DESCRIPTION
**Pull request by yeliang2258**
1. Add RKNN deploy backend support for quantized model
2. Fix the problem that the output will be fused by the quantization pass
3. Update to version 1.0.0

### CODE REVIEW

File: paddle2onnx/mapper/quantize_helper.cc [hunk 1]
```diff
@@ -918,6 +1035,11 @@ void QuantizeModelProcessor::QuantizeInfoBroadcast() {
     } else if (output_quantize_info_iter != helper_->quantize_info.end()) {
       helper_->quantize_info[input_name] = helper_->quantize_info[output_name];
     }
+    if (ConnetToOutput(output_name)) {
```
Comment (jiangjiajun) [inline paddle2onnx/mapper/quantize_helper.cc]: Connect?

File: paddle2onnx/mapper/quantize_helper.h [hunk 2]
```diff
@@ -68,20 +68,31 @@ struct QuantizeModelProcessor {
...
 
   // Read scale file
   void ReadScaleFile(const std::string& scale_file);
 
-  // According to:
+  // Add QDQ for TRT according to:
   // https://github.com/NVIDIA/TensorRT/tree/main/tools/pytorch-quantization/pytorch_quantization/nn/modules
   void AddTrtQDQ();
 
+  void QuantizeInfoReviseForRKNN();
```
Comment (jiangjiajun) [inline paddle2onnx/mapper/quantize_helper.h]: 添加详细的注释，说明为什么RKNN需要增加这个函数

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

File: paddle2onnx/mapper/quantize_helper.cc (hunk 1 above)
Comment (yeliang2258) [reply paddle2onnx/mapper/quantize_helper.cc]: Done

File: paddle2onnx/mapper/quantize_helper.h (hunk 2 above)
Comment (yeliang2258) [reply paddle2onnx/mapper/quantize_helper.h]: 这个函数删除了，和RKNN商议过，Paddle2ONNX提供原始的scale参数，scale信息的处理由RKNN自己来做。

**Pull request merged**